app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -130,15 +200,25 @@ private class HomeTileAdapter(val onUrlEnteredListener: OnUrlEnteredListener, va
             if (hasFocus) {
                 backgroundResource = R.drawable.home_tile_title_focused_background
                 textColor = Color.WHITE
+                menuButton.visibility = View.VISIBLE
+                currentFocusedIndex = position
             } else {
                 backgroundResource = 0
                 textColor = Color.BLACK
+                menuButton.visibility = View.GONE
+                currentFocusedIndex = -1
             }
             titleView.setBackgroundResource(backgroundResource)
             titleView.setTextColor(textColor)
         }
     }
 
+    fun removeItemAtPosition(position: Int) {
+        tiles.removeAt(position)
+        notifyItemRemoved(position)
+        notifyItemRangeChanged(position, itemCount)
```

Comment:
I think this can be removed now that you've removed the long-click listener...maybe?f